src/components/features/ctf-components/ctf-teaser/ctf-teaser-gql.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {
+  CtfTeaser
+} from '@src/components/features/ctf-components/ctf-teaser/ctf-teaser';
+import {
+  useCtfComponentTeaserQuery
+} from '@src/components/features/ctf-components/ctf-teaser/__generated/ctf-teaser.generated';
+
+export interface DefaultQueryParameters {
+  id: any;
+  locale: any;
+  preview: any;
+  sectionType: any
+}
+
+export function CtfTeaserGql(props: DefaultQueryParameters) {
+  const data = useCtfComponentTeaserQuery({
+    id: props.id,
+    locale: props.locale,
+    preview: props.preview,
+  }) as any;
+
+  if (data.data?.teaser) {
+    data.data!.teaser.sectionType = props.sectionType;
```

Comment:
Volgens mij als je data gecheckt hebt met ? in de if, denk ik niet dat die ! er nog bij moet, wel soms?